packages/applicant/src/pages/mandatory-questions/start.page.test.tsx

```diff
@@ -0,0 +1,51 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import MandatoryQuestionsBeforeYouStart, {
+  MandatoryQuestionsProps,
+} from './start.page';
+import { getProps } from 'gap-web-ui';
+import { RouterContext } from 'next/dist/shared/lib/router-context';
+import { createMockRouter } from '../../testUtils/createMockRouter';
+
+function getDefaultProps(): MandatoryQuestionsProps {
+  return {
+    schemeId: '1',
+  };
+}
+
+describe('Mandatory Questions Start', () => {
+  test('should render page', () => {
+    render(
+      <RouterContext.Provider
```

Comment:
could use this helper :
 renderWithRouter(
      <MandatoryQuestionsBeforeYouStart
        {...getPageProps(getDefaultProps)}
      />
      
      from import {
  getPageProps,
  renderWithRouter,
} from '../../../testUtils/unitTestHelpers';

Reply:
not working for some reason so ill leave it